### DISCUSSION TITLE
LocationAwareKafkaLogger logs

### DISCUSSION THREAD

**Discussion by Yedidyah27** [strimzi/strimzi-kafka-operator#5164] [Q&A]
`Consumer clientId=connector-consumer-HCTvsY22kWpekkKP6wWY-g-1244665500-2, groupId=connect-HCTvsY22kWpekkKP6wWY-g-1244665500] Error while fetching metadata with correlation id 1806833 : {HCTvsY22kWpekkKP6wWY-g-1244665500=UNKNOWN_TOPIC_OR_PARTITION}`

We are getting constant logs like the one above for consumers and topics that have been erased. Anyone have an idea how to stop these logs and requests?

**Comment by scholzj** [comment]
That basically just means that the topic does not exist I think.

**Reply by Yedidyah27** [reply]
It was deleted so it shouldnt exist. The connector doesnt exist either but I am still getting these logs all the time

**Reply by scholzj** [reply]
Well, if the topic doesn't exist, this is probably expected then. I do not know why your connector still exists since you didn't provided any details about that.
